src/ext/java/org/opentripplanner/ext/vilkkubikerental/VilkkuBikeRentalDataSourceParameters.java

```diff
@@ -5,23 +5,16 @@
 import org.opentripplanner.updater.vehicle_rental.VehicleRentalSourceType;
 import org.opentripplanner.updater.vehicle_rental.datasources.params.VehicleRentalDataSourceParameters;
 
-public class VilkkuBikeRentalDataSourceParameters extends VehicleRentalDataSourceParameters {
-
-  private final boolean allowOverloading;
-  private final String network;
-
-  public VilkkuBikeRentalDataSourceParameters(
-    String url,
-    String network,
-    boolean allowOverloading,
-    @Nonnull Map<String, String> httpHeaders
-  ) {
-    super(VehicleRentalSourceType.VILKKU, url, httpHeaders);
-    this.network = network;
-    this.allowOverloading = allowOverloading;
-  }
-
-  public String network() {
-    return network;
+public record VilkkuBikeRentalDataSourceParameters(
+  String url,
+  String network,
+  boolean allowOverloading,
+  @Nonnull Map<String, String> httpHeaders
```

Comment:
Make a defensive copy here, this apply to the other records as well witch have collections/maps.